selenium/integration/admin.spec.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { By } from 'selenium-webdriver';
+import { dataSe, buildChromeDriver, loginIfRequired } from '../support/utils';
+import { CONFIGURATION_URL } from '../support/constants';
+
+let driver;
+const TEST_KEY = 'name';
+const TEST_VALUE = 'hello';
+
+describe('Setting and saving preferences', () => {
+  beforeAll(async () => {
+    driver = buildChromeDriver();
+    await loginIfRequired(driver);
```

Comment:
I'm not sure why the cypress test calls this function. it doesn't really use the `CDAP_AUTH_TOKEN` anywhere. Decided to copy it over just in case.